package.json

```diff
@@ -36,41 +47,76 @@
       "email": "joshua.newman@krux.com"
     }
   ],
-  "bugs": "https://github.com/krux/postscribe/issues",
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://github.com/krux/postscribe/blob/master/LICENSE"
-    }
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:krux/postscribe.git"
+  "main": "dist/postscribe.js",
+  "browser": "dist/postscribe.min.js",
+  "engines": {
+    "node": ">=0.10"
```

Comment:
Should we just put `*` here? I don't see much benefit to saying newer than 0.10 since it won't actually be useful inside of node.